components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -156,7 +154,6 @@ export const AppRoutes = () => {
                     <Route path={usagePathMain} exact component={Usage} />
                     <Route path={settingsPathIntegrations} exact component={Integrations} />
                     <Route path={settingsPathNotifications} exact component={Notifications} />
-                    <Route path={settingsPathBilling} exact component={Billing} />
```

Comment:
I think we should have a redirect here, for both `/billing` and settingsPathBilling (if they aren't identical) :+1:

Reply:
More specifically: `/user/billing` -> `/billing` would be nice.
Or, at least have it not stuck in a "Hang in there".